portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/UserManagement.testcase

```diff
@@ -38,12 +38,61 @@ definition {
 	}
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-8143 | Test Summary: Owner can delete users"
-	@ignore = "true"
 	@priority = "5"
 	test ViewOwnerCanDeleteUser {
+		ACNavigation.goToSettings();
+
+		ACSettings.goToUserManagement();
+
+		ACSettings.addUser(userInvite = "user@liferay.com");
+
+		ACUtils.searchBar(searchItem = "user");
+
+		ACSettings.assertUserListEmail(userEmail = "user@liferay.com");
+
+		ACSettings.deleteUser(userDelete = "user@liferay.com");
+
+		AssertTextNotPresent(
+			locator1 = "ACSettings#USER_LIST_EMAIL",
+			value1 = "user@liferay.com");
```

Comment:
Hi @joshitagaki, your code is very good. My suggestion would be for you to do this first part of the creation and deletion using a other user than **TEST**. It could be with `bryan.cheung@faro.io`. What do you think?

Reply:
Thanks Vinicius! Sure I think that's a good idea we can use bryan.cheung@faro.io login.